constants.xml

```diff
@@ -67,6 +67,7 @@
         <NUM_OF_TREEBASED_CHILD_CLUSTERS>5</NUM_OF_TREEBASED_CHILD_CLUSTERS>
         <FETCH_LOOKUP_MSG_MAX_RETRY>3</FETCH_LOOKUP_MSG_MAX_RETRY>
         <MAX_CONTRACT_DEPTH>5</MAX_CONTRACT_DEPTH>
+        <MISORDER_TOLERANCE_IN_PERCENT>2</MISORDER_TOLERANCE_IN_PERCENT>
```

Comment:
The default value can change to 5, since now 2 is too tight and is triggered.

Reply:
ok